crates/bitwarden/src/secrets_manager/secrets/secret_response.rs

```diff
@@ -14,7 +15,6 @@ use crate::{
 #[derive(Serialize, Deserialize, Debug, JsonSchema)]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 pub struct SecretResponse {
-    pub object: String,
```

Comment:
Most likely to keep things consistent. Server includes a `object` field which matched this. But it's not really needed in our case since we've typed everything.

Reply:
Sounds good! thank you.